sale_double_validation/tests/test_double_validation.py

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017-2018 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo.addons.sale.tests.test_sale_common import TestSale
+
+
+class TestSaleDoubleValidation(TestSale):
+
+    def test_one_step(self):
+        so = self.so
+        so.company_id.sudo().so_double_validation = 'one_step'
+        # confirm quotation
+        so.sudo(self.user).action_confirm()
+        self.assertEquals(so.state, 'sale')
+
+    def test_two_steps_under_limit(self):
+        so = self.so
+        so.company_id.sudo().so_double_validation = 'two_step'
+        # confirm quotation
+        so.sudo(self.user).action_confirm()
+        self.assertEquals(so.state, 'sale')
+
+    def test_two_steps_manager(self):
+        so = self.so
+        so.company_id.sudo().so_double_validation = 'two_step'
+        so.company_id.sudo().so_double_validation_amount = 10
+        # confirm quotation
```

Comment:
check that the total amount of the order is what expected (greater/less than `10`)